ckan/views/dataset.py

```diff
@@ -1108,21 +1122,78 @@ def activity(package_type: str, id: str) -> Union[Response, str]:  # noqa
         u'auth_user_obj': g.userobj
     })
     data_dict = {u'id': id}
+    base_limit = int(config.get(u'ckan.activity_list_limit', 30))
+    max_limit = int(config.get(u'ckan.activity_list_limit_max', 100))
+    limit = min(base_limit, max_limit)
+    activity_types = [activity_type] if activity_type else None
+    if after is not None:
+        after = datetime.fromtimestamp(float(after))
```

Comment:
You should handle users (or rather scrapers) passing a non-float here

Reply:
Done